src/Course2/week2/DijkstraAlgo.java

```diff
@@ -0,0 +1,54 @@
+package Course2.week2;
+import java.util.*;
+
+import static java.lang.Integer.min;
+
+public class DijkstraAlgo {
+
+
+     public static int[] Dijkstra(int graph[][], int src){
+
+        int v = graph.length;
+        int[] dist = new int[v];
+
+        Arrays.fill(dist,Integer.MAX_VALUE);
+        dist[src]=0;
+
+        boolean fin[] = new boolean[v];
+
+        for(int count=0;count<v-1;count++){
```

Comment:
nested for loop isn't recommended. change strategy or move the for loops in separate method